api-router.js

```diff
@@ -4,14 +4,38 @@ const calcTxInfo = require('./util/calc-tx-info.js');
 const rskTokenBridgeController = require('./rsk-token-bridge-controller.js');
 const getAddressReport = require('./util/address-report.js');
 const rskActivityReport = require('./util/activity-report/index.js');
+const {
+  verifyParams,
+  readCache,
+  updateCache,
+  getParamValues,
+} = require('./util/functions.js');
 
 const router = express.Router();
 
 router.use(function (req, res, next) {
-  console.log(Date.now());
+  console.log(new Date());
```

Comment:
if you are after human-readable dates, consider using [`Date.toISOString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) instead, because it is also lexical sort friendly, so more common in logs.

Reply:
in my system `new Date()` is printed in the console with purple (probably because it's an object), so it easier to read. However we could use the [morgan logger middleware](https://www.npmjs.com/package/morgan) instead